manifests/crd/v1/pingcap.com_backups.yaml

```diff
@@ -31,10 +40,14 @@ spec:
       jsonPath: .status.backupSizeReadable
       name: BackupSize
       type: string
-    - description: The commit ts of tidb cluster dump
+    - description: The commit ts of the backup
       jsonPath: .status.commitTs
       name: CommitTS
       type: string
+    - description: The log backup truncate until ts
+      jsonPath: .status.truncateUntil
+      name: TruncateUntil
```

Comment:
how about add a `Log` preifix for this status to indicate this is only for backup(mode=log) `LogTruncateUntil `

Reply:
maybe also add for other PiTR fields

Reply:
ok, let me have a look and clarify the name.

Reply:
modify filed name at [this](https://github.com/pingcap/tidb-operator/pull/4648/commits/89b00ea60b80fd3395eec4c75750f1ab1e31baf6)

Reply:
seems you forget update the yaml? this file still named `TruncateUntil `

Reply:
sorry, i forget to fix the kubebuilder's annotation, i fix it [here](https://github.com/pingcap/tidb-operator/pull/4648/commits/944b43b0352d4af11b1779fb56acb995215a2f1f#diff-e171a17263892307a81a6ea400575a37f8ca69d1ff6905462f185c339d118d51R48).